docs/docs/developers/reference/environment_reference/cli_wallet_reference.md

```diff
@@ -41,6 +41,64 @@ You can create arbitrary aliases with the `alias` command. For example `aztec-wa
 
 :::
 
+## Paying Fees
+
+
+import { Why_Fees, CLI_Fees } from '/components/snippets';
+
+<Why_Fees />
+
+Below are all the payment methods available to pay transaction fees on Aztec, starting with the simplest.
+
+### Sponsored Fee Paying Contract
+
+This is a special type of FPC (explained [below](#fee-paying-contract)) that can be used to pay for account deployment and regular txs.
+Note: the contract alias below is that of a sponsored fee paying contract.
+
+#include_code fpc-sponsored yarn-project/cli-wallet/test/flows/sponsored_create_account_and_mint.sh bash
+
+:::Note
+In the sandbox, the sponsored FPC address is printed at the end of its initial logs.
+:::
+
+### Fee Juice from Sandbox Test accounts
+
+In the sandbox pre-loaded test accounts can be used to cover fee juice when deploying contracts.
+
+First import them:
+
+#include_code import-test-accounts yarn-project/cli-wallet/test/flows/basic.sh bash
+
+Then use the alias (test0, test1...) when paying in fee juice. Eg to create accounts:
+
+#include_code declare-accounts yarn-project/end-to-end/src/guides/up_quick_start.sh bash
+
+### Bridging fee juice
+
+First register an account and bridge fee juice to it via minting:
+
+#include_code bridge-fee-juice yarn-project/cli-wallet/test/flows/create_account_pay_native.sh bash
+
+You'll have to wait for two blocks to pass for bridged fee juice to be ready on Aztec.
+For the sandbox you do this by putting through two arbitrary transactions. Eg:
+
+#include_code force-two-blocks yarn-project/cli-wallet/test/flows/create_account_pay_native.sh bash
+
+Now the funded account can deploy itself with the bridged fees:
```

Comment:
add that here you are claiming the bridged fee juice AND doing your main tx in one go - this is not clear atm.